grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -313,6 +313,11 @@ class StockMovementService {
         if (requisitionItem.modificationItem) {
             requisitionItem = requisitionItem.modificationItem
```

Comment:
I know this is not part of the current PR but can someone explain what's happening here. This looks a bit dangerous. Are we really making picklist changes on the modification item? I would feel more comfortable maintaining the picklist on the parent requisition item rather than one of its children.

Reply:
It has been implemented by you. Modification items have different quantity than original one, why should picklist be maintained on the parent item then?

Reply:
@jmiranda I believe there is no better way to exaplin the issue rather than conversation between developer and RequisitionItem so you did it perfectly :) And yes, you're right, the way it's implemented right now is a bit complicated and could/should be refactored one day.